src/Localization/en/MaterializedViewsBuilder.json

```
@@ -44,5 +44,28 @@
-    "WarningBannerOnDelete": "After deprovisioning the Materializedviews Builder, your materializedviews will not be updated with new source changes anymore. Materializedviews builder is compute in your account that performs read operations on source collection for any updates and applies them on materialized views as per the materializedview definition."
+    "WarningBannerOnDelete": "After deprovisioning the Materializedviews Builder, your materializedviews will not be updated with new source changes anymore. Materializedviews builder is compute in your account that performs read operations on source collection for any updates and applies them on materialized views as per the materializedview definition.",
+    "GlobalsecondaryindexesBuilderDescription": "Provision a Global secondary indexes builder cluster for your Azure Cosmos DB account. Global secondary indexes builder is compute in your account that performs read operations on source collection for any updates and updates the global secondary indexes as per the Global secondary indexes definition.",
+    "GlobalsecondaryindexesBuilder": "Global secondary indexes Builder",
+    "LearnAboutGlobalSecondaryIndexes": "Learn more about global secondary indexes.",
+    "GlobalsecondaryindexesDeprovisioningDetailsText": "Learn more about global secondary indexes.",
+    "GlobalsecondaryindexesBuilderPricing": "Learn more about Global secondary indexes pricing.",
+    "GlobalsecondaryindexesCreateMessage": "Global Secondary Indexes Builder resource is being created.",
+    "GlobalsecondaryindexesCreateInitializeMessage": "Global secondary indexes Builder resource will be provisioned.",
+    "GlobalsecondaryindexesCreateSuccesseMessage": "Global secondary indexes Builder resource provisioned.",
+    "GlobalsecondaryindexesCreateFailureMessage": "Global secondary indexes Builder resource provisioning failed.",
+    "GlobalsecondaryindexesUpdateMessage": "Global secondary indexes Builder resource is being updated.",
+    "GlobalsecondaryindexesUpdateInitializeMessage": "Global secondary indexes Builder resource will be updated.",
+    "GlobalsecondaryindexesUpdateSuccesseMessage": "Global secondary indexes Builder resource updated.",
+    "GlobalsecondaryindexesUpdateFailureMessage": "Global secondary indexes Builder resource updation failed.",
+    "GlobalsecondaryindexesDeleteMessage": "Global secondary indexes Builder resource is being deleted.",
+    "GlobalsecondaryindexesDeleteInitializeMessage": "Global secondary indexes Builder resource will be deleted.",
+    "GlobalsecondaryindexesDeleteSuccesseMessage": "Global secondary indexes Builder resource deleted.",
+    "GlobalsecondaryindexesDeleteFailureMessage": "Global secondary indexes Builder resource deletion failed.",
+    "GlobalsecondaryindexesCostText": "Hourly cost of the Global secondary indexes Builder resource depends on the SKU selection, number of instances per region, and number of regions.",
```

Is this true? I thought the builder was only ever provisioned in one region? Should we update this text?

```suggestion
    "GlobalsecondaryindexesCostText": "Hourly cost of the global secondary indexes builder resource depends on the SKU selection, number of instances per region, and number of regions.",
```